attributes/default.rb

```diff
@@ -43,13 +43,12 @@
 # Node Virtualenv Path
 default['cfncluster']['node_virtualenv_path'] = "#{node['cfncluster']['system_pyenv_root']}/versions/#{node['cfncluster']['python-version']}/envs/#{node['cfncluster']['node_virtualenv']}"
 # Intel Packages
-default['cfncluster']['psxe']['version'] = '2019.5'
-default['cfncluster']['intelhpc']['version'] = '2018.0-*.el7'
-default['cfncluster']['intelpython2']['version'] = '2019.4'
```

Comment:
`intelpython2` omitted on purpose. See here: https://software.intel.com/content/www/us/en/develop/articles/intel-distribution-for-python-release-notes.html